python-data-version-control/src/prepare.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import pandas as pd
+
+folders_to_labels = {"n03445777": "golf ball", "n03888257": "parachute"}
+
+
+def get_filelist_and_labels(source_path, target_folders):
```

Comment:
It feels a little odd to me that you're taking `target_folders` as an argument here. It seems like you would just check that the folder for an image is in `folders_to_labels` (on line 13) are and not worry about taking any argument.

I guess there might be some architectural reason for doing something like this in a real-world project, but for this example, I think we can simplify it.

```suggestion
def get_filelist_and_labels(source_path):
```